### DISCUSSION TITLE
Unable to login to account after moving database to new server

### DISCUSSION THREAD

**Discussion by IlyaPetrovM** [directus/directus#13075] [Archive]
Hello! 
I'm trying to move my database from one server (Debian, Mariadb) to another (Windows  10, MariaDb). 
I created new database in Win, and then export database usind phpmyadmin in Linux server. 

All data imported successfully. But I can't log in using old account. Only new admin account works.

How to fix that?

**Comment by rijkvanzanten** [comment]
Is your Directus instance connected to the correct database? It sounds to me like you're connecting into a new Directus installation rather than using the database that holds the old dump 🤔

**Reply by IlyaPetrovM** [reply]
Well I have noticed that I used SSO for auth in Linux, but in windows I want to use login and password (I will run it locally).

And in Linux server I can't enter to system with my own password (without SSO)!

**Reply by IlyaPetrovM** [reply]
I found the issue. It is the same problem https://github.com/directus/directus/issues/9331
But
I want to switch off SSO in Win server and chage all passwords to default value (I don't want old passwords).
Is it possible?

**Comment by IlyaPetrovM** [comment]
Well I had the same issue: https://github.com/directus/directus/issues/9331

I set `provider = Default` in phpmyadmin for all users and set them default password usind directus.
